daft/io/config.py

```diff
@@ -2,8 +2,7 @@
 
 import json
 
-from daft.daft import PyIOConfig as IOConfig
-from daft.daft import PyS3Config as S3Config
+from daft.daft import IOConfig, S3Config
```

Comment:
Note: I had to do this because it seems Sphinx really doesn't like aliasing. The docstrings were never getting picked up, instead only ever showing up as `alias of PyIOConfig`.

Tried proposed solutions here to no avail: [StackOverflow qn](https://stackoverflow.com/questions/38765577/overriding-sphinx-autodoc-alias-of-for-import-of-private-class)